coss/club/models.py

```diff
@@ -7,6 +7,41 @@
 from location_field.models.plain import PlainLocationField
 
 
+class ClubRegistration(models.Model):
+    CLUB_TYPE = (
+        ('opensource', 'Open Source'),
+        ('webliteracy', 'Web Literacy'),
+        ('womenchildren', 'Women & Children'),
+    )
+    CLUB_STATUS = (
+        ('new', 'New Club'),
+        ('existing', 'Existing Club'),
+        ('notsure', 'Not Sure'),
+    )
+    CLUB_PRIVACY = (
+        ('public', 'Public Visibility'),
+        ('open', 'Anyone Can Join'),
+        ('private', 'Private & Closed'),
+    )
+
+    club_type = models.CharField(max_length=40, choices=CLUB_TYPE, default='opensource')
+    club_status = models.CharField(max_length=40, choices=CLUB_STATUS, default='new')
+    club_name = models.CharField(max_length=40)
+    club_description = RichTextField()
+    club_location = models.CharField(max_length=255)
+    club_place = models.CharField(max_length=255)
+    club_privacy = models.CharField(max_length=40, choices=CLUB_PRIVACY, default='public')
+    fullname = models.CharField(max_length=255)
+    location = models.CharField(max_length=255)
+    email = models.EmailField(max_length=255)
+    student_year = models.CharField(max_length=255, blank=True)
```

Comment:
Probably we can drop this completely and aim for a minimum viable product. We can add more fields when we have the skeleton so that more people can work in parallel. I would say if this requires some effort, let's drop it and open an issue instead.